tasks/db_deploy/migrate_db.py

```diff
@@ -22,27 +22,27 @@ def perform_migration(current_schema_version: int, config: Dict[str, str]) -> No
     """
```

Comment:
Is it possible to migrate the DB to the new 'shared' location? That would require pointing the lambda at both locations.

Reply:
not sure about this. What does the 'shared' location refer to?